### DISCUSSION TITLE
Action Buttons left side

### DISCUSSION THREAD

**Discussion by DZ4-Miguel** [Power-Components/livewire-powergrid#725] [Q&A]
Hi friends, is there an elegant way to display the action buttons on the left side of the table?

**Comment by BenOussama180** [comment]
![image](https://user-images.githubusercontent.com/81111919/202707296-29eb47cc-4b3d-436e-a71a-2d15cbd5e080.png)
this is how i worked arounnd it, using Blade::render() i rennder inside is a button with wire:click=emit for a modal using WireElements modal package

**Reply by BenOussama180** [reply]
btwn i am rendering 3 buttons in that column thats why it is looking a bit hectic haha

**Comment by marvin-aleman** [comment]
Same here. Is there a simple way of doing this? Ben's solution defeats the purpose of using PowerGrid.

**Reply by luanfreitasdev** [reply]
I see no problem using directly on the columns. But I would let Blade Render use component instead of adding all html inside addColumn.

**Reply by marvin-aleman** [reply]
Since there was no alternative I took a better look at Ben's html and realized it had a lot of stuff I didn't need. It was easier to adapt than expected.
It'd be cool if there was an easy way of doing this, like a method on the Button class. But I guess this will do.
![imagen](https://user-images.githubusercontent.com/40923344/218461568-5ef9039d-94a5-4541-b615-3de9f248672f.png)
Next step will be making components for the buttons.
